Do i still need a smartwatch (#268)
mauroservienti/milestone.github.io: _posts/do-I-still-need-a-smartwatch.md

---
layout: post
header_image: /img/posts/do-I-still-need-a-smartwatch/header.jpg
title: "Do I still need a smart watch?"
author: Mauro Servienti
synopsis: "I feel misplaced sometimes. People around me, apparently, happily buy a lot of things I find completely useless. Every time I look at my wish list I only find good reasons to not buy something. Am I missing something crucial? What do I really need?"
tags:
- personal
- self-organization
---

> I'm surrounded by people spending money, as if there is no tomorrow, for things I find completely useless.

Reviewer: ```suggestion
I'm surrounded by people spending money as if there is no tomorrow, for things I find completely useless.

```

Even if I find most of the stuff people buy useless, I'm not immune to the marketing sirens. I'm continuously tempted and maintain a long Amazon wish list to accumulate stuff.

I regularly go through my list and evaluate items. I use a pros/cons approach to determine if I need them. I remove them from the wish list all the time. The only things that survive are LEGO bricks.

One of the things that survived the most on the wish list is an activity tracking device. In my case, I'm keeping an eye on many different Garmin devices. I'm a happy Garmin Edge 830 and a relatively old Forerunner 920XT owner. They served me well over the years. Being a bike rider, my primary use for the Edge is turn-by-turn navigation. I cannot care less about stats, speed, or other numbers the device spits. I sit down at my desk, use Garmin Connect to plan a ride, upload the ride to the device, and finally ride.

I use the Forerunner for a similar job. When swimming, I follow a workout made of lengths, pauses, drills, and some drowning too ;-)
Until last year, the Forerunner was a handy companion. I preload workouts to the device and follow them in the pool.

> This year I changed the pool where I go swimming, and in this pool, there are workouts available for free. They are printed, and plastic coated. Swimmers can bring them in the water, and the plastic coat, when wet sticks well to the wall.

Reviewer: Just trying to remove some commas

```suggestion
This year I changed the pool where I go swimming, and in this pool, there are workouts available for free. They are printed, and plastic coated. Swimmers can bring them in the water and the plastic coat sticks well to the wall when wet.
```

> At the same time, all decent polls have one of these "clocks":

Reviewer: ```suggestion
At the same time, all decent pools have one of these "clocks":

```

![pool clock](image to add)

The only thing a Forerunner substitute could do more for me is zone-based training. For those of up you unaware of that, it's the possibility of training within a specific heart rate range. The device vibrates if your heart rate goes above or below the defined range, and the swimmer knows that they need to push more or less.

Is that useful? I sincerely don't know.

## What am I really looking for?

The reality is that I'm looking for a coach. And I've been trying to solve the problem through a device for a while, and I failed. It was probably expected.

I need someone or something to tell me: do this to improve that. I have a feeling I reached the limits of self-training.

> So far, I never realized that was my essential need, and I always watched at devices like cool gadgets that come with tons of features. The thing is that all those features were making me blind. They were like a sugar rush, making me excited for no real good reasons.

Reviewer: ```suggestion
So far, I never realized that was my essential need, and I always looked at devices like cool gadgets that come with tons of features. The thing is that all those features were making me blind. They were like a sugar rush, making me excited for no good reason.
```

## Conclusion 

> This short post is written from a personal point of view but under the hood, what I'm hinting at is: when designing things, be careful not to hide the important stuff in the "sugar rush," unless the essential element isn't there at all. By the way, when it comes to swimming, it seems that my colleague [Christian](https://twitter.com/cquirosj) is a perfect partner in crime. He has so many ideas for improvements!

Reviewer: ```suggestion
This short post is written from a personal point of view but under the hood, what I'm hinting at is: when designing things, be careful not to hide the important stuff in the "sugar rush," unless the essential element isn't there at all. By the way, when it comes to swimming, it seems that my colleague [Christian](https://twitter.com/cquirosj) is a perfect partner in crime. He has so many ideas for improvements! Sometimes the best technology is just another person that shares your interest :)

```
